oidc-controller/api/templates/verified_credentials.html

```diff
@@ -233,6 +233,13 @@ <h1 v-if="mobileDevice" class="mb-3 text-start fw-bolder fs-1">
                     value="{{url_to_message}}"
                   >
                   </textarea>
+
+                  <!-- Add a button that disconnects the websocket -->
+                  <button
+                    class="btn btn-primary mt-4"
+                    v-on:click="socket.disconnect()"
+                    title="Disconnect Websocket"
+                  >DEBUG Disconnect Web Socket</button>
```

Comment:
To explicitly disconnect (shows up in the 10 clicks debug area). Manual disconnection will not trigger a reconnect but we'll handle and manually reconnect below.